specs/decisionmatrix/vsphere.yaml

```diff
@@ -160,4 +160,58 @@ rows:
     max_size: 8192
     priority: 0
     thin_provisioning: true
-    drive_type: "eagerzeroedthick"
\ No newline at end of file
+    drive_type: "eagerzeroedthick"
+  - instance_type: "*"
+    instance_min_drives: 1
+    instance_max_drives: 12
+    region: "*"
+    min_size: 32
+    max_size: 100
+    priority: 0
+    thin_provisioning: true
+    drive_type: "lazyzeroedthick"
+  - instance_type: "*"
+    instance_min_drives: 1
+    instance_max_drives: 12
+    region: "*"
+    min_size: 150
+    max_size: 500
+    priority: 0
+    thin_provisioning: true
+    drive_type: "lazyzeroedthick"
+  - instance_type: "*"
+    instance_min_drives: 1
+    instance_max_drives: 12
+    region: "*"
+    min_size: 500
+    max_size: 1024
+    priority: 0
+    thin_provisioning: true
+    drive_type: "lazyzeroedthick"
+  - instance_type: "*"
+    instance_min_drives: 1
+    instance_max_drives: 12
+    region: "*"
+    min_size: 1024
+    max_size: 2048
+    priority: 0
+    thin_provisioning: true
```

Comment:
nit: although not used, let's set thin_provisioning: false for all lazyzeroedthick

Reply:
ok